src/modules/calculate.js

```diff
@@ -0,0 +1,44 @@
+const numberClick = (e) => {
```

Comment:
- [ ] Kindly consider importing `operate.js` file here. there is no doubt that your app is working fine but we must adhere to the guidelines and requirements mentioned in the project requirements. 
please consider following the guidelines mentioned in the project requirements. you may need to follow the flow as described below:

Project requirements
To help you concentrate on how to work with React events, we have developed the math logic you need to make the Calculator component work. You can use the files [calculate.js](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js) and [operate.js](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js).

Create a new directory called logic inside your project
Copy the files calculate.js and operate.js into your logic/ directory
Install [big.js](https://www.npmjs.com/package/big.js) using NPM
Analyze the files calculate.js and operate.js, be sure you understand what they do
Import calculate.js in your Calculator.js component
Implement click event handlers using the math logic from calculate.js and operate.js in your React component
The goal is to make all calculator buttons work according to their functionality e.g. the - button should subtract a given value